src/httpbis/index.ts

```diff
@@ -112,11 +114,13 @@ export function deriveComponent(component: string, params: Map<string, string |
             if (!params.has('name')) {
                 throw new Error('@query-param must have a named parameter');
             }
-            const name = (params.get('name') as BareItem).toString();
+            const name = decodeURIComponent((params.get('name') as BareItem).toString());
+            console.log(name);
+            console.log(searchParams);

```

Comment:
Please can you remove the console calls 🙏

Reply:
Oops, artefacts of debugging